packages/stacked/README.md

```diff
@@ -458,14 +458,14 @@ class InformationService with ReactiveServiceMixin { //1
 }
 ```
 
-Easy peasy. This service can now be listened too when any of the properties passed into the `listenToReactiveValues` is changed. So how do listen to these values? I'm glad you asked. Lets move onto the `ReactiveViewModel`.
+Easy peasy. This service can now listen too when any of the properties passed into the `listenToReactiveValues` is changed. So how do you listen to these values? I'm glad you asked. Let's move onto the `ReactiveViewModel`.
 
 ### Reactive View Model
 
-This ViewModel extends the `BaseViewModel` and adds an additional function that allows you to listen to services that are being used in the model. There are two thing you have to do to make a ViewModel react to changes in a service.
+This ViewModel extends the `BaseViewModel` and adds a function that allows you to listen to services that are being used in the model. There are two things you have to do to make a ViewModel react to changes in a service.
 
 1. Extend from `ReactiveViewModel`.
-2. Implement reactiveServices getter that return a list of reactive services.
+2. Implement reactive services getter that returns a list of reactive services.
```

Comment:
please make this one word again using camel case, it refers to the code to be used. Change returns back to return

Reply:
But `returns` is correct here grammatically